### PR TITLE
Release: Fix setup.py for webui

### PR DESCRIPTION
**Pull request by bari12**

### CODE REVIEW

File: setup_webui.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
     from setuputil import get_rucio_version
 
 name = 'rucio-webui'
-packages = ['rucio', 'rucio.web', 'rucio.web.ui', 'rucio.web.ui.common']
+packages = ['rucio', 'rucio.web', 'rucio.web.ui']
```
Comment (bziemons) [inline setup_webui.py]: `packages = ['rucio', 'rucio.web', 'rucio.web.ui', 'rucio.web.ui.flask', 'rucio.web.flask.common']` ?

Comment (bari12) [reply setup_webui.py]: Yes, you are right. Not sure why we didn't add this in the first place. Will add.

Comment (bziemons) [reply setup_webui.py]: probably because this setup.py is long forgotten :D

**Review by bziemons — APPROVED** [review]

File: setup_webui.py (hunk 1 above)
Comment (bari12) [reply setup_webui.py]: Actually, `rucio.web.flask.common ` doesn't exist :-)

Comment (bziemons) [reply setup_webui.py]: I missed the `ui`, correct. Should've been `rucio.web.ui.flask.common`

**Pull request merged**